pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -120,6 +120,9 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 				if !ok {
 					return nil, fmt.Errorf("plugin %v does not extend score plugin", sc.Name)
 				}
+				if f.pluginNameToWeightMap[p.Name()] == 0 {
```

Comment:
We need to do an exist check because in reality Name() could return something different from the name set in the config.

Reply:
If it does not exist, the value of `weight` will be 0, so it is same to check whether it is 0.

Reply:
ok, sounds good then.